README.md

```diff
@@ -123,7 +123,7 @@ https://getcomposer.org/doc/01-basic-usage.md
 
 ## XDebug
 
-You need to run `sudo ifconfig lo0 alias 10.254.254.254` before you can use xdebug.
+You need to run `sudo ifconfig lo0 alias 10.254.254.254` before xdebug connections will work. This is usually required each time you log-in to your development machine, but is safe to run periodically.
```

Comment:
Out of interest, could this be run as part of the docker start script rather than you having to run it each time?

Reply:
I did think about that, but as it requires `sudo` access to run it may be awkward to run it automatically in the Makefile.